lib/generators/logidze/install/functions/logidze_logger.sql

```diff
@@ -89,11 +89,25 @@ CREATE OR REPLACE FUNCTION logidze_logger() RETURNS TRIGGER AS $body$
       changes := '{}';
 
       IF (coalesce(current_setting('logidze.full_snapshot', true), '') = 'on') THEN
-        changes = hstore_to_jsonb_loose(hstore(NEW.*));
+        BEGIN
+          changes = hstore_to_jsonb_loose(hstore(NEW.*));
+        EXCEPTION
+          WHEN NUMERIC_VALUE_OUT_OF_RANGE THEN
```

Comment:
It's also possible to use the `OTHERS` condition (_The special condition name OTHERS matches every error type except QUERY_CANCELED and ASSERT_FAILURE._), don't really sure.

Reply:
I think, NUMERIC_VALUE_OUT_OF_RANGE is enough here. Unless we know which else could go wrong during hstore conversion.